.config/playwright-setup.js

```diff
@@ -86,33 +85,14 @@ function spawn_gradio_app(app, port, verbose) {
 				res(_process);
 
 				if (!verbose) {
-					_process.stdout.destroy();
-					_process.stderr.destroy();
+					_process.stdout.off("data", std_out);
+					_process.stderr.off("data", std_out);
 				}
```

Comment:
I didn't realise that `destroy` actually destroyed the stdio functionality of the child process, I thought it just removed listeners. This was causing any python process that tried to print to the terminal to throw an exception. I've changed this to just unlisten rather than destroy.